frontend/src/metabase/metadata/pages/DataModel/types.ts

```diff
@@ -0,0 +1,20 @@
+import type {
+  DatabaseId,
+  FieldId,
+  SchemaName,
+  TableId,
+} from "metabase-types/api";
+
+export interface RouteParams {
```

Comment:
Can we use `type =` here?

```suggestion
export type ParsedRouteParams = {
```

Reply:
ac11cfc6e6e3a45b793b4bbc7195d57155cbbc37